lib/matplotlib/font_manager.py

```diff
@@ -566,17 +566,16 @@ def createFontList(fontfiles, fontext='ttf'):
         if fontext == 'afm':
             try:
                 fh = open(fpath, 'rb')
-            except:
+            except OSError:
                 verbose.report("Could not open font file %s" % fpath)
                 continue
             try:
-                try:
```

Comment:
This probably dates back to before you could actually have an `except` and `finally` on the same `try`.